src/client/components/dashboard/dashboard_robot/model.ts

```diff
@@ -12,45 +15,75 @@ import PenaltyReason = message.input.GameState.Data.PenaltyReason
 import Phase = message.input.GameState.Data.Phase
 
 export class DashboardRobotModel {
+
+  // Parameters that influence the display
   @observable public camera: Transform
-  @observable public battery: number
   @observable public ballColor: string
-  @observable public ballPosition: Vector2
   @observable public ballSightColor: string
-  @observable public ballWorldPosition: Vector2
-  @observable public behaviourState: State
-  @observable public gameMode: Mode
-  @observable public gamePhase: Phase
-  @observable public id: number
-  @observable public kickTarget: Vector2
   @observable public kickTargetColor: string
-  @observable public lastCameraImage: number
-  @observable public lastSeenBall: number
-  @observable public lastSeenGoal: number
-  @observable public lastSeenObstacle: number
-  @observable public penaltyReason: PenaltyReason
   @observable private robot: RobotModel
   @observable public robotBorderColor: string
   @observable public robotColor: string
-  @observable public robotHeading: Vector2
-  @observable public robotPosition: Vector2
   @observable public textColor: string
+
+  // Parameters from the network
+  // The timestamp of the last message from the robot
   @observable public time: number
+
+  // The id number of the robot
+  @observable public id: number
+
+  // The name of the role the robot is executing
+  @observable public roleName: string
+
+  // Battery as a value between 0 and 1 (percentage)
+  @observable public battery: number
+
+  // The voltage of the battery
   @observable public voltage: number
 
-  constructor(robot: RobotModel, opts: Partial<DashboardRobotModel>) {
+  // The state of the behaviour as an enum
+  @observable public behaviourState: State
+
+  // The robots position and heading and associated covariance
+  @observable public robotPosition: Vector3 // x,y,theta
```

Comment:
Not for this PR, but we might want to introduce a semantic class for this structure if it becomes super common.